packages/manager/modules/enterprise-cloud-database/src/create/create.html

```diff
@@ -366,13 +366,14 @@ <h5>
                 >
                 </enterprise-cloud-database-payment-info-component>
                 <div class="row">
-                    <div
-                        class="col-md-6"
-                        data-ng-if="$ctrl.order"
-                        data-ovh-contracts="$ctrl.order.contracts"
-                        data-full-text="false"
-                        data-ovh-contracts-validated="$ctrl.contractsAccepted"
-                    ></div>
+                    <div class="col-md-6" data-ng-if="$ctrl.order">
```

Comment:
Hm if `$ctrl.order` is null, then the `class="row"` right above will be incorrect so you can remove the row altogether

Reply:
You're right! We will improve it.